dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -180,5 +180,12 @@ JWK_RETRIEVE_FAILED=CWWKS1747E: A JSON Web Key (JWK) was not returned from the U
 JWK_RETRIEVE_FAILED.explanation=A JSON Web Key cannot be returned from the specified URL. The URL might not be valid, the URL might not be configured to return a JWK, the response from the URL might be empty, or an unknown error occurred.
 JWK_RETRIEVE_FAILED.useraction=Verify that the URL is formatted correctly and specifies a location that is capable of returning JSON Web Keys. Check the status code and content of the response for more information.
 
+USERINFO_RETREIVE_FAILED=CWWKS1748E: An attempt to retrieve UserInfo from from the URL [{0}] failed.  The response status was [{1}] and the content returned was [{2}].
```

Comment:
The object precedes the descriptor; for example:
An attempt to retrieve UserInfo from from the [{0}] URL failed.

Reply:
It won't be apparent what the object is,  so the descriptor needs to lead. 
For example, in the old existing message 
OIDC_CLIENT_BAD_PARAM_COOKIE=CWWKS1745E: The WASOidcCode cookie 4q5890uawerlkfjaxlwu4oiqfrn;adpzov;k/...

is more understandable than "the 4q5890uawerlkfjaxlwu4oiqfrn;adpzov;k/ WASOidcCode cookie...
I'll update to make it a little more declarative though.